projects/igniteui-angular/src/lib/grids/pivot-grid/pivot-grid-dimensions.ts

```diff
@@ -46,6 +47,11 @@ export interface IPivotDateDimensionOptions {
 export class IgxPivotDateDimension implements IPivotDimension {
     /** Enables/Disables a particular dimension from pivot structure. */
     public enabled = true;
+    
+    /**
+     * Gets/Sets data type
+     */
+    public dataType?: GridColumnDataType;
```

Comment:
I think for the date dimensions we should set the column type to be Date by default, so that the user does not have to explicitly set it every time. In most cases you would always use this for date values anyway.